codecov.yml

```diff
@@ -0,0 +1,6 @@
+coverage:
+  status:
+    project:
+      default:
+        target: 60%
+        threshold: 0.5%
```

Comment:
Does it make sense to add additional rules?
```
comment:
  require_changes: true
```
That would do the following
`Comments will now only post when coverage changes. Furthermore, if a comment already exists, and a newer commit results in no coverage change for the entire pull, the comment will be deleted.`

Reply:
Not sure if we need comments at all. If the coverage is to low you will see it in the check list

Reply:
I think this refers to the comment the bot posts which at the moment we always get. So, it will only be posted when the coverage changes. But I am fine either way